paddlets/datasets/tsdataset.py

```diff
@@ -2071,7 +2071,7 @@ def dtypes(self) -> pd.Series:
             type_list.append(pd.DataFrame(self._static_cov, index=[0]).dtypes)
         return pd.concat(type_list)
 
-    def sort_columns(self, ascending: bool = True):
+    def sort_columns(self, ascending: Optional[bool] = True):
```

Comment:
Line2074 ascending should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional